Update fleets in Getting started guides (#2027)
balena-io/docs: shared/getting-started/projectSpecific/_default.md

> To give your device a public URL, access the device page, and choose the _Public Device URL_ toggle. You may also activate this for many devices in your fleet at the same time via the _Actions_ menu in the fleets dashboard.

Reviewer: ```suggestion
To give your device a public URL, access the device page, and choose the _Public Device URL_ toggle. You may also activate this for many devices in your fleet at the same time via the _Actions_ menu on the device list within a fleet.
```
